edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/config/StoreLimits.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core.Config
+{
+    using System;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.Devices.Edge.Util.Json;
+    using Newtonsoft.Json;
+
+    public class StoreLimits : IEquatable<StoreLimits>
+    {
+        [JsonConstructor]
+        public StoreLimits(long maxSize)
+            : this(maxSize, Option.None<int>())
+        {
+        }
+
+        public StoreLimits(long maxSize, Option<int> checkFrequency)
+        {
```

Comment:
Add a comment here stating why this constructor and the checkFrequency property exist in this class so that no one accidentally ends up consuming it for other purposes outside of test